python/featomic/tests/clebsch_gordan/rotations.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, max_angular: int, angles: Sequence[float] = None):
         # Randomly generate Euler angles between 0 and 2 pi if none are provided
         if angles is None:
             angles = np.random.uniform(size=(3)) * 2 * np.pi
+            angles[1] = angles[1] / 2
```

Comment:
why?

Reply:
From Michelangelo: "since the theta  angles in the Euler is always between 0 and pi"

Reply:
Maybe a quick comment saying that angles[1] should be between 0 and pi? Or 

```
psi = 2 * np.pi * np.random.uniform(size=1)
theta = np.pi * np.random.uniform(size=1)
phi = 2 * np.pi * np.random.uniform(size=1) 
```

Reply:
I don't see a comment here?